settings.py

```diff
@@ -994,6 +994,9 @@
     'pillowtop': ['localhost']
 }
 
+DATADOG_API_KEY = None
+DATADOG_APP_KEY = None
```

Comment:
nope, the management command gets called on one of the servers in the cluster via fabric